### ISSUE TITLE
Unexpectedly removed single-line comments inside an empty block

### ISSUE THREAD

**Issue by hstarorg** [ThisIsManta/vscode-stylus-supremacy#15]
I hope it don‘t remove that. A config property will very good.

**Comment by ThisIsManta** [comment]
Would you kindly share an example of the removed comments?

This is hard for me because the AST parser never returns a comment, hence, I need to preserve comments myself, which is painful.

**Comment by hstarorg** [comment]
Like this: 
```stylus
.page-input-demo {
  .sa-input {
    // margin-bottom: 10px;
    // border: 1px solid red;
  }
}
```
After format: 
```stylus
.page-input-demo {
  .sa-input {
  }
}
```
BTW: If the code is 
```stylus
.page-input-demo {
  .sa-input {
    margin-bottom: 10px;
    // border: 1px solid red;
  }
}
```
The bug can't be reproduced.

**Comment by ThisIsManta** [comment]
Please use multi-line comments (/* */) as a workaround.

**Comment by ThisIsManta** [comment]
Just want to tell you that I'll be busy until next week. Please bear with me.

**Comment by hstarorg** [comment]
@ThisIsManta Thanks, work is more important. And it's not a serious bug.

**Comment by nachocab** [comment]
Any update on this? It would be nice to have an option to preserve comments

**Comment by ThisIsManta** [comment]
I'll take care of this today. Stay tuned!

**Comment by ThisIsManta** [comment]
This is fixed in [version 2.5.0](https://github.com/ThisIsManta/stylus-supremacy/releases/tag/v2.5.0).
If you're using **Atom** as a code editor, please wait a couple days to get the new version.

Cheers!

**Comment by nachocab** [comment]
Great, thanks! I'm using VSCode
